code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -29,6 +29,24 @@
 	var/backstab_bonus = 30
 	var/light_on = FALSE
 	var/brightness_on = 5
+	var/destabilizer_type = /obj/item/projectile/destabilizer
+
+/obj/item/twohanded/kinetic_crusher/premium
```

Comment:
crushers are already the meta. Why are you buffing that meta?

Reply:
It has really small chance to drop

Reply:
If it's powerful then make it rare is why ninja is such shit balance as well. It's a lazy excuse for balancing just hiding something behind RNG. At least PerishedFraud's lavaland PRs are trying to fix the dumb RNG, this takes it the opposite direction.